tools/pckgen

```diff
@@ -298,6 +299,7 @@ rpm_spec_prepare() {
     sed -i "s/<CAS_NAME>/$CAS_NAME/g" "$RPM_SPECS_DIR/$CAS_NAME.spec"
     sed -i "s/<CAS_VERSION>/$CAS_VERSION/g" "$RPM_SPECS_DIR/$CAS_NAME.spec"
     sed -i "s/<CAS_LICENSE_NAME>/$CAS_LICENSE_NAME/g" "$RPM_SPECS_DIR/$CAS_NAME.spec"
+    sed -i "s|<CAS_MODULES_DIR>|$CAS_MODULES_DIR|g" "$RPM_SPECS_DIR/$CAS_NAME.spec"
```

Comment:
Why `|`?

Reply:
Sed can take virtually any character as a separator. In this case we are replacing a directory path that already contains slashes, so it would interfere with sed separators if we'd used slashes here also.